### PR TITLE
Event app namespacing fixes

### PR DESCRIPTION
**Pull request by martypowell**
- Enable the event services and constants to be shared between the events page app and events widget
- Fix everything that was broke when we changed that

### CODE REVIEW

File: js/apps/events-page/app.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
 
-	angular.module('eventsPageApp', ['ngRoute', 'ngSanitize']);
+	angular
+		.module('eventsPageApp', 
+			[
+				'dataServices',
+				'events',
+				'ngRoute', 
+				'ngSanitize'
+			]);
```
Comment (mxsnyder) [inline js/apps/events-page/app.js]: Don't forget to include ngAria in there.

File: js/apps/events-page/constants.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+(() => {
 	'use strict';
 
+	const app = angular.module('events', []);
+
 	var constants = {
-		baseUrl: 'https://testservices.bcpl.info',
-		// baseUrl: 'http://oit226471:1919',
+		// baseUrl: 'https://testservices.bcpl.info',
+		baseUrl: 'http://oit226471:1919',
```
Comment (mxsnyder) [inline js/apps/events-page/constants.js]: You'll want to move the comment back.

File: mockups/html/index.html [hunk 3]
```diff
@@ -453,7 +453,7 @@ <h2 class="branding">Featured Events</h2>
 							</div>
 						</header>
 						<div id="events-target">
-							<div ng-app="eventsPageApp">
+							<div ng-app="featuredEventsWidget">
```
Comment (mxsnyder) [inline mockups/html/index.html]: What do you think of featuredEventsWidgetApp as the module name?

**Comment by martypowell** [comment]
@mxsnyder all comments addressed, ready for another look

File: package.json [hunk 4]
```diff
@@ -4,6 +4,7 @@
   "description": "Web feature assets for BCPL",
   "main": "index.js",
   "dependencies": {
+    "angular-aria": "^1.6.9",
```
Comment (mxsnyder) [inline package.json]: This should be a devdependency since we use the CDN versions of those files (maybe not forever, but at the moment we do).

**Pull request merged**